composer.json

```diff
@@ -22,7 +22,7 @@
         }
     },
     "require": {
-        "php": ">=5.6.5"
+        "php": ">=5.6.5 || >=7.0"
```

Comment:
Yes, it should.

See also https://github.com/symfony/symfony/blob/master/composer.json for example.

Why did you change this @LauLaman ?